packages/react/src/BitteWalletContext.tsx

```diff
@@ -73,21 +74,21 @@ export const BitteWalletContextProvider: React.FC<ContextProviderType> = ({
 
   const { setupBitteWalletSelector, registerWalletAccountsSubscriber, connectWalletSelector, pollForWalletConnection, disconnectFromWalletSelector, signMessage } = BitteWalletAuth;
 
-  const setupMbWallet = async (): Promise<WalletSelectorComponents> => {
-    const isOnlyMbWallet = !!onlyMbWallet || !!(additionalWallets && additionalWallets.length > 0);
+  const setupBitteWallet = async (): Promise<WalletSelectorComponents> => {
+    const isOnlyBitteWallet = !!onlyBitteWallet || !!(additionalWallets && additionalWallets.length > 0);
 
     return await setupBitteWalletSelector(
       callbackUrl,
-      isOnlyMbWallet,
+      isOnlyBitteWallet,
       selectedNetwork,
       selectedContract,
-      isOnlyMbWallet ? { additionalWallets } : undefined,
+      isOnlyBitteWallet ? { additionalWallets } : undefined,
       successUrl, failureUrl,
     );
   };
```

Comment:
I think we could live without having the wallet name everywhere. Like why not just `setupWallet`? People know its the bitte wallet from the import.